templates/company/about.html

```diff
@@ -0,0 +1,464 @@
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">
+              <a href="/solutions/ubuntu-os">Build with confidence. Build on Ubuntu.</a>
+            </h3>
+          </div>
+          <div class="col-6">
+            <p>Bring harmony to your stack from the desktop to the edge with Ubuntu, the OS that innovators have in common.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">
+              <a href="/solutions/infrastructure">Cloudify your data center</a>
+            </h3>
+          </div>
+          <div class="col-6">
+            <p>Gain more control and optimize costs with open source infrastructure.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">
+              <a href="/solutions/secure-open-source">Secure your open source supply chain</a>
+            </h3>
+          </div>
+          <div class="col-6">
+            <p>Keep your developers productive and streamline compliance.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">
+              <a href="/data">Run your databases reliably, across clouds</a>
+            </h3>
+          </div>
+          <div class="col-6">
+            <p>Accelerate time to market with secure and supported open source database software.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">
+              <a href="/solutions/ai">Simplify your enterprise AI journey</a>
+            </h3>
+          </div>
+          <div class="col-6">
+            <p>Enjoy the freedom of open source, with the best support and economics for AI projects.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  
+  <section class="p-section">
```

Comment:
Please just use the deep strip here, this section is adding additional spacing 
design: 
![image](https://github.com/user-attachments/assets/63aafc42-e3f2-483a-bde9-a3dce1fdfbf1)
demo:
![Screenshot from 2024-10-25 14-39-40](https://github.com/user-attachments/assets/52241247-7cec-4071-8181-ac896c6019da)
![Screenshot from 2024-10-25 14-39-44](https://github.com/user-attachments/assets/f7d6eb97-66e2-4deb-a128-4aa348f5fdbb)